### PR TITLE
feat(helpdesk): Web helpdesk

### PR DESCRIPTION
**Pull request by kksteini**
## Helpdesk web

A new helpdesk site that serves up questions and answer based on organizations and relevant support categories

### Entrypoint to helpdesk
image

### Category view
image

### Category view with question
image

### Search
image

Coming soon

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by siggifv — APPROVED** [review]
Reviewed changes in `libs/api/domains/content-search/`. LGTM

**Review by stjanilofts — APPROVED** [review]

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -365,7 +365,7 @@ jobs:
           PUBLISH: 'true'
 
       - name: Comment on PR
-        if: needs.pre-checks.outputs.LABELED_FOR_DEPLOY == 'true'
+        # if: needs.pre-checks.outputs.LABELED_FOR_DEPLOY == 'true'
```
Comment (sindrig) [inline .github/workflows/push.yml:368]: hmm why?

Comment (stjanilofts) [reply .github/workflows/push.yml:368]: Not sure 🤷‍♂️
I think it had something to do with the deploy-feature? This should probably be changed back I guess.

**Review by sindrig — APPROVED** [review]
LGTM

**Pull request merged**